packages/flutter/test/widgets/selectable_region_test.dart

```diff
@@ -654,6 +654,54 @@ void main() {
     });
   });
 
+  testWidgets('Can extend MultiStaticSelectableSelectionContainerDelegate', (WidgetTester tester) async {
+    SelectedContent? content;
+    final FocusNode focusNode = FocusNode();
+
+    // Inserts a new line between selected content of children selectables.
+    final ColumnSelectionContainerDelegate selectionDelegate = ColumnSelectionContainerDelegate();
+
+    addTearDown(focusNode.dispose);
+    addTearDown(selectionDelegate.dispose);
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: SelectableRegion(
+          onSelectionChanged: (SelectedContent? selectedContent) => content = selectedContent,
+          focusNode: focusNode,
+          selectionControls: materialTextSelectionControls,
+          child: Center(
+            child: SelectionContainer(
```

Comment:
Super nit: I wonder if it's better to put the SelectionContainer above the Center? I'm guessing most people will use this to cover everything under as single SelectableRegion. Or is there any benefit to keeping this as-is?

Reply:
That makes sense to me, updated.